target/linux/ath79/dts/ar9342_ubnt_nanobeam-ac.dts

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9342_ubnt_wa.dtsi"
+
+/ {
+	compatible = "ubnt,nanobeam-ac", "ubnt,wa", "qca,ar9342";
+	model = "Ubiquiti NanoBeam AC (WA)";
+
+	gpio-leds {
```

Comment:
@TobleMiner Could you please change this to generic name `leds` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist)

>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation

While at it, could you please add `aliases` node to `ar9342_ubnt_wa.dtsi` with description of LED aliases as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) as for example done in [ar9342_ubnt_xm.dtsi](https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/ar9342_ubnt_xw.dtsi#L12)?

>  If it's possible try to dedicate some of the LEDs for system status indication in [diag.sh](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/diag.sh) ([example]())

Ideally you want to do the same as it's done in `ar71xx` currently.

Reply:
You are absolutly right about the "gpio-leds" node. I've fixed that in the latest version.

I don't think adding a aliases node in this PR is the proper way to introduce that change though. This should be put into a separate ubnt_wa devicetree cleanup PR.

Also LEDs for diagnostics will most probably never be a thing on ubnt_wa devices. None of them do have any software controllable LEDs other than the rssi indicator LEDs. I'd not like abusing them for diagnostics since this kind of dual-use makes distinguishing between the different modes of operation (rssi indicator vs diagnsotics) harder.

Reply:
> I don't think adding a aliases node in this PR is the proper way to introduce that change though. This should be put into a separate ubnt_wa devicetree cleanup PR.

If the change is made in the separate commit, I don't see a problem with it being in the same PR.

> Also LEDs for diagnostics will most probably never be a thing on ubnt_wa devices.

Same situation is on ubnt_xm devices. We (ab)use link4(rssi3 on WA) for that purpose. I think, that some feedback, that the device is (re)booting or in failsafe mode could be useful.

Anyway, this support is not mandatory, it's optional.